### PR TITLE
Add support for multiple video tracks

### PR DESCRIPTION
**Pull request by kiddkaffeine**

### CODE REVIEW

**Comment by flibitijibibo** [comment]
Functionally it makes sense, but let's simplify this by disallowing video streams where size does not match. Theorafile could possibly support it separately, but similar to audio we should assert that the formats should be identical; you should only need to change streams for non-technical reasons (i.e. language changes). For different formats, different files should be used - trying to support multiple formats in one container is too error-prone given the XNA spec's constraints.

**Comment by kiddkaffeine** [comment]
Removed the logic for resizing VideoPlayer mid-stream.  This may be inadvisable anyway; changing video tracks midstream (even when everything is generally almost identical) generally seems to cause momentarily compression artifacting.

File: src/Media/Xiph/Video.cs [hunk 1]
```diff
@@ -173,6 +173,13 @@ public void SetAudioTrackEXT(int track)
 			}
 		}
 
+		public void SetVideoTrackEXT(int track)
+		{
+			if (theora != IntPtr.Zero) {
```
Comment (flibitijibibo) [inline src/Media/Xiph/Video.cs:178]:
```suggestion
			if (theora != IntPtr.Zero)
			{
```

**Review by flibitijibibo — APPROVED** [review]
Just a newline and this is good

File: src/Media/Xiph/Video.cs (hunk 1 above)
Comment (kiddkaffeine) [reply src/Media/Xiph/Video.cs]: Updated.

**Comment by flibitijibibo** [comment]
If I don't rebuild tonight I'll likely get this merged on the 30th. If anyone wants to check those sizeof values between now and then, that'll help too.

**Pull request merged**